spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -210,6 +210,7 @@ public Object deserialize(byte[] bytes) throws ZkMarshallingError {
 
 	private static final Set<Object> KAFKA_PRODUCER_PROPERTIES = new SetBuilder()
 			.add(BinderPropertyKeys.MIN_PARTITION_COUNT)
+			.add(BinderPropertyKeys.REQUIRED_GROUPS)
```

Comment:
I am trying to understand the role of `requiredGroups` in Kafka binder as it is considered as a `supported` property. Could you clarify?